surveysvc.go

```diff
@@ -1,192 +0,0 @@
-package main // import "github.com/onsdigital/rm-survey-service"
-
-import (
-	"database/sql"
-	"net/http"
-	"os"
-	"strconv"
-	"time"
-
-	"go.uber.org/zap"
-
-	"github.com/cloudfoundry-community/go-cfenv"
-	"github.com/labstack/echo"
-	"github.com/labstack/echo/middleware"
-	"github.com/onsdigital/rm-survey-service/models"
-)
-
-const serviceName = "surveysvc"
-const timeFormat = "2006-01-02T15:04:05Z0700"
-
-var logger *zap.Logger
-
-func init() {
-	logger, _ = zap.NewProduction()
-	defer logger.Sync()
-}
-
-func main() {
-	port := ":8080"
-	dataSource := "postgres://postgres:password@localhost/postgres?sslmode=disable"
-	appEnv, err := cfenv.Current()
-
-	if err == nil {
-		ps := appEnv.Port
-		port = ":" + strconv.FormatInt(int64(ps), 10)
-		postgresServer, err := appEnv.Services.WithTag("postgresql")
-		logInfo("Found Cloud Foundry environment")
-
-		if err == nil {
-			uri, found := postgresServer[0].CredentialString("uri")
-
-			if found {
```

Comment:
Then it's bad news :-( Rob S actually added that code, I'm blaming him! I'll add an `else` with some logging.